trlx/model/accelerate_base_model.py

```diff
@@ -97,44 +96,21 @@ def tokenize(self, text: Iterable[str]):
         return self.tokenizer(
             text,
             truncation=True,
-            padding="max_length",
-            max_length=self.config.train.input_size,
+            max_length=self.config.seq_length,
             return_tensors="pt",
         )
 
-    def act(
-        self, data: PromptBatch
-    ) -> Tuple[
-        TensorType["chunk_size", "input_length"],
-        TensorType["chunk_size", "gen_size"],
-        Iterable[str],
-    ]:
-        query_tensors = data.tokens.to(
-            self.accelerator.device
-        )  # [B, N] #TODO(dahoas): This may need to be changed
+    def generate(self, input_ids, attention_mask=None, **kwargs):
```

Comment:
I noticed we are no longer loading the model into accelerate at init time. If we just want to do large model (>20B) inference do we still need to load with accelerate?